lms/djangoapps/certificates/queue.py

```diff
@@ -214,6 +214,18 @@ def add_cert(self, student, course_id, course=None, forced_grade=None, template_
         Returns the newly created certificate instance
         """
 
+        if not hasattr(course_id, 'ccx'):
```

Comment:
This seems to be the way we usually check if something is a CCX. I feel it would be better to keep it consistent.

e.g. https://github.com/edx/edx-platform/blob/8ae92901ef882ee18323eb87447f4090a0098716/lms/djangoapps/courseware/access.py#L84